app/models/validations/sales/property_validations.rb

```diff
@@ -7,4 +7,13 @@ def validate_postcodes_match_if_discounted_ownership(record)
       record.errors.add :ppostcode_full, I18n.t("validations.property.postcode.must_match_previous")
     end
   end
+
+  def validate_bedsit_number_of_beds(record)
+    return if record.proptype.blank? || record.beds.blank?
+
+    unless record.proptype != 2 || record.beds <= 1
```

Comment:
I try to avoid using unless not, it might be easier to read if you change this into an if statement 👀

Reply:
Thanks @kosiakkatrina! Changed now :)